src/buildVariables.js

```diff
@@ -99,12 +96,7 @@ const buildUpdateVariables = (resource, aorFetchType, params, queryType) =>
     }, {});
 
 const buildCreateVariables = (resource, aorFetchType, params, queryType) => {
-    return Object.keys(params.data).reduce((acc, key) => {
-        return {
-            ...acc,
-            [key]: params.data[key],
-        };
-    }, {});
+    return params.data;
```

Comment:
This is actually all, I changed, the other changes, came from prettier-auto-format.